packages/apps-config/src/api/spec/index.ts

```diff
@@ -96,6 +97,8 @@ const spec: Record<string, OverrideBundleDefinition> = {
   'dock-testnet': dock,
   'dotmog-node': dotmog,
   dusty4: dusty,
+  eave: eave,
+  'dawn':eave,
```

Comment:
```suggestion
  eave: eave,
  dawn: eave,
```

It may complain about it being alphabetical here - do probably should just move dawn.